lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -733,12 +733,12 @@ func (h *StorageHandlersImpl) StatPath(params storage.StatPathParams) middleware
 
 }
 
-// ListImages returns a list of images in a store
+// GetImageStorageUsage returns images storage usage for this VCH
 func (h *StorageHandlersImpl) GetImageStorageUsage(params storage.GetImageStorageUsageParams) middleware.Responder {
 	op := trace.NewOperationFromID(context.Background(), params.OpID, "GetImageStorageUsage(%s)", params.StoreName)
 	defer trace.End(trace.Begin("GetImageStorageUsage", op))
 
-	result, err := h.imageCache.DataStore.GetImageStorageUsage(op)
+	result, err := h.imageCache.DataStore.GetImageStorageUsage(op, params.StoreName)
```

Comment:
Does StoreName equal VCH UUID? I think it is the VCH name.

Reply:
Yes, it is VCH UUID